book/novel_applications/payment/private_payment.html

```diff
@@ -0,0 +1,449 @@
+                            <i id="print-button" class="fa fa-print"></i>
+                        </a>
+
+                    </div>
+                </div>
+
+                <div id="search-wrapper" class="hidden">
+                    <form id="searchbar-outer" class="searchbar-outer">
+                        <input type="search" id="searchbar" name="searchbar" placeholder="Search this book ..." aria-controls="searchresults-outer" aria-describedby="searchresults-header">
+                    </form>
+                    <div id="searchresults-outer" class="searchresults-outer hidden">
+                        <div id="searchresults-header" class="searchresults-header"></div>
+                        <ul id="searchresults">
+                        </ul>
+                    </div>
+                </div>
+
+                <!-- Apply ARIA attributes after the sidebar and the sidebar toggle button are added to the DOM -->
+                <script>
+                    document.getElementById('sidebar-toggle').setAttribute('aria-expanded', sidebar === 'visible');
+                    document.getElementById('sidebar').setAttribute('aria-hidden', sidebar !== 'visible');
+                    Array.from(document.querySelectorAll('#sidebar a')).forEach(function(link) {
+                        link.setAttribute('tabIndex', sidebar === 'visible' ? 0 : -1);
+                    });
+                </script>
+
+                <div id="content" class="content">
+                    <main>
+                        <h1 id="private-payment"><a class="header" href="#private-payment">Private Payment</a></h1>
+<p><img src="../../assets/images/private_payment.png" alt="Private payment" /></p>
+<h2 id="the-why"><a class="header" href="#the-why">The Why?</a></h2>
+<h3 id="introduction"><a class="header" href="#introduction">Introduction</a></h3>
+<p>Efficient payments are a backbone of human societies. They enable exchange of value for goods and services, facilitate social functioning (participating in activities e.g., going to the restaurant), and support economic growth. Without them, commerce would grind to a halt, financial stability would be undermined, and access to essential services would be severely limited, potentially leading to economic stagnation and social disruption.</p>
+<h3 id="blockchains-as-a-payments-infrastructure"><a class="header" href="#blockchains-as-a-payments-infrastructure">Blockchains as a Payments Infrastructure</a></h3>
+<p>Since the very first blockchain, Bitcoin, payments have been the flagship use case. With further developments like Ethereum and alternative L1's (Polygon, Tron, Near) and innovations like rollups (Base, Starknet, Scroll), payments have been made faster, cheaper, and more powerful (with additional functionalities e.g., through smart contracts).</p>
+<div class="table-wrapper"><table><thead><tr><th>Blockchain</th><th>Bitcoin</th><th>Ethereum</th><th>Polygon</th><th>Base</th></tr></thead><tbody>
```

Comment:
I am not entirely sure the given chart presents the message that you have given above. 

<img width="605" alt="SCR-20240913-oemk" src="https://github.com/user-attachments/assets/72a06c3b-5e36-41cf-9038-aff66c11df2f">

Ethereum here seems to suffice considering that it has a volume superior to all the others combined.

I can rework the introduction to shorten + focus on stables.

Reply:
Nonetheless, I can rework the introduction: shorten + focus on stables

Reply:
I would say, this is not correct. Ethereum has the highest volume with an average transaction of around 100k. But this is not our use case. I would look at the number of transactions. Here Celo seems to be interesting with 16$ average per transaction, also PoS and Tron. 

<img width="1333" alt="Screenshot 2024-09-11 at 18 10 55" src="https://github.com/user-attachments/assets/e451f0ae-eee9-40a6-bbe8-7f155b1ba4f4">

Those seem to be more human to human payments rather than bigger DeFi transactions as on Ethereum.